apps/10address-book-ajax/script.js

```diff
@@ -1,5 +1,55 @@
 'use strict';
 
+// object containing list of users for ajax call on the entire userlist
+var userList = {
+  // use forEach to loop through the user list
+  success: function(users) {
+    // for each specific user...
+    users.forEach(function(user) {
+      // ...create a string containing the following data and store it in variable "str"
+      var str = 
+      ('<tr>' + 
+      '<td>' + user.id + '</td>' + 
+      '<td>' + user.first_name + '</td>' + 
+      '<td>' + user.last_name + '</td>' +
+      // this data ensures a user's unique id is attached to their specific link in the DOM
+      '<td>' + '<a href="#" data-id="' + user.id + '">view</a></td>' +
+      '</tr>');
+      // append data to the tbody element
+      $('tbody').append(str);
+    });
+  }
+}
+
+// object containing specific user for a specific ajax call
+var specificUser = {
+  success: function(user) {
+    // when a specific user is called, store their data in variable "str"...
+    var str = 
+    ('<div id="details">' + 
+      '<h3>' + user.first_name + user.last_name + '</h3>' + 
```

Comment:
I would recommend putting a space ` ' ' `  in between `user.first_name` and `user.last_name` fam.